static/index.html

```diff
@@ -3,38 +3,41 @@
 	<head>
 		<meta charset="UTF-8">
 		<title>Greets Graf</title>
-		<link rel="stylesheet" type="text/css" href="style.css">
-		<script type="text/javascript" src="utils.js"></script>
-		<script type="text/javascript" src="script.js"></script>
+		<link rel="stylesheet" href="style.css">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<script src="utils.js"></script>
+		<script src="script.js"></script>
 	</head>
 
 	<body>
 		<div id="main-wrapper">
 			<div id="main-content">
 				<h1>Let's fill some greetings</h1>
-				Demofinder 2000:
+
+				<label for="in_prod">Demofinder 2000:</label>
 				<div id="prod_search" class="autocomplete" style="width: 320px;">
 					<input type="text" id="in_prod" style="width: 100%;" placehoder="Prod name here...">
 				</div>
-				<span style="color: #777;">(or Pouet prod id: <input type="text" style="width: 6em;" id="in_prod_id"/><button onclick="loadProd()">Load</button>)</span>
+				<span style="color: #777;">(or <label>Pouet prod id: <input type="text" style="width: 6em;" id="in_prod_id"></label> <button onclick="loadProd()">Load</button>)</span>
 
-				<br/>
+				<br>
 
 				<div id="prod_container" style="display: none;">
 					<div id="prod"></div>
-					<strong>Greetings:</strong><br/>
+
+					<strong>Greetings:</strong><br>
 					<div id="greets_list"></div>
-					Add new greeting to group:
+
+					<label for="greet_add_group">Add new greeting to group:</label>
 					<span class="autocomplete" style="width: 256px;">
 						<input type="text" value="" style="width: 100%;" id="greet_add_group" placeholder="Group name">
 					</span>
-					Note:
-						<input type="text" value="" placeholder="e.g. 13:37 or nfo" id="greet_add_note">
-						<button id="greet_add_button" disabled>add</button>
+					<label>Note: <input type="text" value="" placeholder="e.g. 13:37 or nfo" id="greet_add_note"></label>
```

Comment:
This label also contains the input itself? Is this correct?

Reply:
Yes, that is correct and intentional. This way it is not necessary to reference the input field by ID because the connection is implicit.

See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

I didn't fit for the autocomplete input field, though.

Reply:
Got it. Thank you!